scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl

```diff
@@ -219,6 +227,61 @@ workflow CNVSomaticPairWorkflow {
             gatk_docker = gatk_docker,
             disk_space_gb = plot_normal_disk
     }
+
+    if (is_run_oncotator) {
+        call Oncotate.CNVOncotateCalledSegments as OncotateCalledCNVWorkflow {
+            input:
+                 called_file = CallCopyRatioSegmentsTumor.called_copy_ratio_segments,
+                 oncotator_docker = oncotator_docker
+        }
+    }
+
+    output {
+        File modeled_segments_tumor = ModelSegmentsTumor.modeled_segments
```

Comment:
Don't forget the outputs of PreprocessIntervals, CollectCountsTumor/Normal, and CollectAllelicCountsTumor/Normal.

Reply:
Done